Dockerfile.rhel7

```diff
@@ -9,11 +9,11 @@ COPY --from=builder /go/src/github.com/openshift/cluster-kube-apiserver-operator
 COPY --from=builder /go/src/github.com/openshift/cluster-kube-apiserver-operator/bindata/bootkube/config /usr/share/bootkube/manifests/config/
 COPY --from=builder /go/src/github.com/openshift/cluster-kube-apiserver-operator/bindata/bootkube/manifests /usr/share/bootkube/manifests/manifests/
 COPY --from=builder /go/src/github.com/openshift/cluster-kube-apiserver-operator/bindata/bootkube/scc-manifests /usr/share/bootkube/manifests/manifests/
-COPY --from=builder /go/src/github.com/openshift/cluster-kube-apiserver-operator/vendor/github.com/openshift/api/apiserver/v1/apiserver.openshift.io_apirequestcount.yaml /usr/share/bootkube/manifests/manifests/
+COPY --from=builder /go/src/github.com/openshift/cluster-kube-apiserver-operator/vendor/github.com/openshift/api/apiserver/v1/zz_generated.featuregated-crd-manifests.yaml /usr/share/bootkube/manifests/manifests/zz_generated.featuregated-crd-manifests.yaml
```

Comment:
This replacement isn't correct, you needed to add the path segment, not just a replace here

Reply:
I updated the values based on https://github.com/openshift/cluster-kube-apiserver-operator/commit/a04fc2619c915312b1a054934ab159eb8670d99d, but the jobs still appear to be failing. [Image builds and pushes](https://gcsweb-ci.apps.ci.l2s4.p1.openshiftapps.com/gcs/test-platform-results/pr-logs/pull/openshift_cluster-kube-apiserver-operator/1696/pull-ci-openshift-cluster-kube-apiserver-operator-release-4.16-e2e-gcp-operator-single-node/1800707298469875712/artifacts/build-logs/) were fine.